### ISSUE TITLE
Dash glyph seems to not have a proper width

### ISSUE THREAD

**Issue by aperezdc** [vernnobile/commeFont#5]
Typing a dash in front of any other character causes the dash and the character to overlap. The cursor does _not_ advance after typing it, which suggests that the dash glyph not have a proper width assigned.

![Typing “-d -D" in a GTK+ font selection dialog](https://cloud.githubusercontent.com/assets/723451/21769883/86d711de-d689-11e6-826e-e985e806e6b2.png)

**Comment by fayad** [comment]
Comme has the potential to be a great linux font even better than Noto sans. But this bug is a serious problem. 

Between what is the font in display there @aperezdc ?

**Comment by jakepogo** [comment]
Size and family of "Comme" doesn't matter, and IE for some reason sizes it correctly but ONLY IE Firefox, Chrome, etc all size it with what appears to be 0 width which makes whatever the next character you add appear on top of the hyphen.

**Comment by vzajcic** [comment]
This still hasn't been addressed? Someone closed their own report for the same issue saying: "Turns out it's a problem with the browser, not the font, because it works in IE and on default Android browser."

This is not a valid thing to say because I've never seen a similar issue on a font I've used, and these minor glitches are honestly the only reason why it isn't widely used IMHO.

@vernnobile thanks for making this font, I'm only sad you're no longer working on it.

**Comment by howdev** [comment]
2019 still no news about this issue? Cannot use this font since all the filenames with hyphen is unreadable

**Comment by fayad** [comment]
The font has not been updated after Vernon's passing, as you can see from the repository's activity.
Anyway happy to see the interest here.

**Comment by howdev** [comment]
yes. Very sad incident of Vernon. But much thankful for Vernon's contribution to the community for great fonts ! The best humanist fonts.
